distributed/scheduler.py

```diff
@@ -569,7 +584,7 @@ def identity(self) -> dict:
             **self.extra,
         }
 
-    def _to_dict_no_nest(self, *, exclude: "Container[str]" = ()) -> dict:
+    def _to_dict_no_nest(self, *, exclude: Container[str] = ()) -> dict[str, Any]:
```

Comment:
The thorough review ends here. Beyond this point it's either auto-tweaks by pyupgrade or fixes to keep things working.